connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerGroupMember.java

```diff
@@ -105,7 +105,7 @@ public WorkerGroupMember(DistributedConfig config,
 
             this.metrics = new Metrics(metricConfig, reporters, time, metricsContext);
             this.retryBackoffMs = config.getLong(CommonClientConfigs.RETRY_BACKOFF_MS_CONFIG);
-            this.metadata = new Metadata(retryBackoffMs, config.getLong(CommonClientConfigs.METADATA_MAX_AGE_CONFIG),
+            this.metadata = new Metadata(retryBackoffMs, retryBackoffMs, config.getLong(CommonClientConfigs.METADATA_MAX_AGE_CONFIG),
```

Comment:
I know the scope of the KIP was limited to the Java clients and not delving into anything with respect to Connect or the Streams framework, but in this case, if we set the `retry.backoff.ms` equal to `retry.backoff.max.ms` what is the behavior in the `ExponentialBackoff` class? Ultimately the `Metadata` portion of this will affect the Connect framework, so I'm trying to understand what will be the impact of setting both to be the same.

Reply:
Yes. If we set `retryBackoffMs` and `retryBackoffMaxMs` to the same value, `Metadata` will have a static backoff.